tools/iframe-test.html

```diff
@@ -0,0 +1,32 @@
+<!-- this file can be loaded directly from a browser and used to test embedding groups and projects within iframes
+see https://github.com/DemocracyLab/CivicTechExchange/issues/978 
+change groups/inframe to projects/inframe below to test embedding a project
+This if for testing without using iframe resizer
+ -->
+<!DOCTYPE html>
+<html lang="en" style="height: 100%">
+  <head>
+    <meta charset="utf-8" />
+    <title>iframe test</title>
+  </head>
+  <body>
+    <div style="width: 100vw;">
```

Comment:
Same width concern as above. My thinking here is, we should have these test frames be as close to drop-in examples as possible, so a little polish that may not be required for internal dev isn't going to be wasted.

Reply:
Happy to make the test-frames good examples, but this is the doc we are going to make available to people who want to try it: https://docs.google.com/document/d/1Uq5vLXrNzIEaXkpEw3X7K5DHvzETnPXKevCYG-O85KM/edit#heading=h.28dw2j9tlcg9